common/src/main/java/com/sonicether/soundphysics/config/SoundReflectivityConfig.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Map;
+
+public class SoundReflectivityConfig extends CommentedPropertyConfig {
+
+    private Map<String, Float> soundReflectivities;
+
+    public SoundReflectivityConfig(Path path) {
+        super(new CommentedProperties(false));
+        this.path = path;
+        reload();
+    }
+
+    @Override
+    public void load() throws IOException {
+        super.load();
+
+        Map<String, Float> map = createDefaultMap();
+
+        for (Map.Entry<String, String> entry : properties.entrySet()) {
+            String key = entry.getKey();
+            float value;
+            try {
+                value = Float.parseFloat(entry.getValue());
+            } catch (Exception e) {
+                Loggers.warn("Failed to set allowed sound entry {}", key);
+                continue;
+            }
+            ResourceLocation resourceLocation;
+            try {
+                resourceLocation = new ResourceLocation(key);
+            } catch (Exception e) {
+                Loggers.warn("Failed to set allowed sound entry {}", key);
+                continue;
+            }
+
+            if (!resourceLocation.getNamespace().equals(AudioChannel.CATEGORY_VOICECHAT)) {
+                logIfUnknownSound(resourceLocation);
+            }
+
+            map.put(resourceLocation.toString(), value);
+        }
+
+        soundReflectivities = ConfigUtils.sortMap(map);
+
+        saveSync();
+    }
+
+    private void logIfUnknownSound(ResourceLocation resourceLocation) {
+        try {
+            SoundEvent soundEvent = BuiltInRegistries.SOUND_EVENT.get(resourceLocation);
+            if (soundEvent == null) {
+                Loggers.log("Unknown sound in allowed sound config: {}", resourceLocation);
+            }
+        } catch (Exception e) {
+            Loggers.warn("Failed to parse allowed sound entry {}", resourceLocation, e);
+        }
+    }
+
+    @Override
+    public void saveSync() {
+        properties.clear();
+
+        properties.addHeaderComment("Sound specific absorption multipliers");
+
+        for (Map.Entry<String, Float> entry : soundReflectivities.entrySet()) {
+            properties.set(entry.getKey(), String.valueOf(entry.getValue()));
+        }
+
+        super.saveSync();
+    }
+
+    public Map<String, Float> getMap() {
+        return soundReflectivities;
+    }
+
+    public float getValue(String soundEvent) {
+        return soundReflectivities.getOrDefault(soundEvent, 1.0F);
+    }
+
+    public Map<String, Float> createDefaultMap() {
+        Map<String, Float> map = new HashMap<>();
+        for (SoundEvent event : BuiltInRegistries.SOUND_EVENT) {
+            map.put(event.getLocation().toString(), 1.0F);
+        }
+
+        map.put(SoundEvents.LIGHTNING_BOLT_THUNDER.getLocation().toString(), 2.5F);
+        map.put(SoundEvents.GENERIC_EXPLODE.getLocation().toString(), 2.5F);
+        SoundEvents.GOAT_HORN_SOUND_VARIANTS.forEach(r -> map.put(r.key().location().toString(), 1.5F));
```

Comment:
Why should these sounds have more reverb?
I'm generally not really seeing the point of having different reflectivities for sounds. 
Also sounds like lightning already have reverb kind of built in to the sound itself, so having a reflectivity of 0 for thunder would make more sense, I think.